packages/loader/container-loader/src/container.ts

```diff
@@ -397,6 +398,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private loaded = false;
     private _attachState = AttachState.Detached;
 
+    private readonly storageServiceAdapter: ContainerStorageAdapter;
```

Comment:
based on the usage, I think you can remove getter and simply have

public readonly storage: IDocumentStorageService;

i.e. exact type of object stored here is not interesting (not used)

Reply:
Removed